tracer/test/test-applications/integrations/Samples.Kafka/Program.cs

```diff
@@ -122,6 +125,53 @@ await Task.WhenAny(
                 Task.Delay(TimeSpan.FromSeconds(5)));
         }
 
+        private static async Task TryProduceWithNullTopicPartition(string topic, ClientConfig config)
+        {
+            // NOTE: you'll see an Null Ref exception, it is expected, but it shouldn't be in OUR logs
```

Comment:
FWIW, this is an interesting point that I think we (I) often overlook - just because the nullable annotations in an instrumented method mean the method itself will throw, _we_ still need to be defensive, because our code runs prior to the method body